lmdeploy/pytorch/engine/logits_process.py

```diff
@@ -34,6 +34,15 @@ class FusedLogitsProcessor(LogitsWarper):
     def __init__(self, sampling_param: SamplingParam):
         self.sampling_param = sampling_param
 
+    def _repetition_penalty(self, input_ids: torch.LongTensor,
```

Comment:
Does `input_ids` include the history tokens?

Reply:
Yes, all history tokens